__init__.py

```diff
@@ -203,83 +195,22 @@ def check_wallpaper_path(self, wallpaper):
-        }
-        if notification_message not in self.notifications_model:
-            self.notifications_model.append(notification_message)
-            self.gui["notifcation_counter"] = len(self.notifications_model)
-            self.gui["notification"] = notification_message
-            time.sleep(2)
-            self.bus.emit(Message("homescreen.notification.show"))
-
-    def handle_clear_notification_data(self, message):
-        """ Clear Pop Notification """
-        notification_data = message.data.get("notification", "")
-        self.notifications_storage_model.append(notification_data)
-        for i in range(len(self.notifications_model)):
-            if (
-                self.notifications_model[i]["sender"] == notification_data["sender"]
-                and self.notifications_model[i]["text"] == notification_data["text"]
-            ):
-                if not len(self.notifications_model) > 0:
-                    del self.notifications_model[i]
-                    self.notifications_model = []
-                else:
-                    del self.notifications_model[i]
-                break
+    # Manage notifications widget
 
-        self.gui["notification_model"] = {
-            "storedmodel": self.notifications_storage_model,
-            "count": len(self.notifications_storage_model),
-        }
-        self.gui["notification"] = {}
-
-    def handle_clear_delete_notification_data(self, message):
-        """ Clear Pop Notification & Delete Notification Data """
-        notification_data = message.data.get("notification", "")
-        for i in range(len(self.notifications_model)):
-            if (
-                self.notifications_model[i]["sender"] == notification_data["sender"]
-                and self.notifications_model[i]["text"] == notification_data["text"]
-            ):
-                if not len(self.notifications_model) > 0:
-                    del self.notifications_model[i]
-                    self.notifications_model = []
-                else:
-                    del self.notifications_model[i]
-                break
-
-    def handle_clear_notification_storage(self, _):
-        """ Clear All Notification Storage Model """
-        self.notifications_storage_model = []
-        self.gui["notification_model"] = {
-            "storedmodel": self.notifications_storage_model,
-            "count": len(self.notifications_storage_model),
-        }
+    def handle_notification_widget_update(self, message):
+        notifcation_count = message.data.get("notification_counter", "")
+        self.log.info("Notifications Counter Changed")
+        self.log.info(notifcation_count)
+        self.gui["notifcation_counter"] = notifcation_count
+        self.bus.emit(Message("ovos.notification.api.request.storage.model"))
+
+    def handle_notification_storage_model_update(self, message):
+        notification_model = message.data.get("notification_model", "")
```

Comment:
Can a docstring be added here to explain what this does (specifically thinking what does this do vs the above method)

Reply:
Doc strings has been added